lib/ansible/executor/task_executor.py

```diff
@@ -518,7 +518,11 @@ def _execute(self, variables=None):
         if '_variable_params' in self._task.args:
             variable_params = self._task.args.pop('_variable_params')
             if isinstance(variable_params, dict):
-                raise AnsibleError("Using a variable for a task's 'args' is not allowed as it is unsafe, facts can come from untrusted sources.")
+                if C.INJECT_FACTS_AS_VARS:
+                    display.warning("Using a variable for a task's 'args' is unsafe in some situations "
+                                    "(see https://docs.ansible.com/ansible/devel/reference_appendices/faq.html#argsplat-unsafe)")
```

Comment:
Do we need to change this to `/latest/` after we release?

Reply:
Ideally it'd be `$current_release`, but that's not variable-ized in our current doc builds TMK, so devel seems like the safest place to have it at the moment. I don't want to add a bunch more doc hacking to the RM branch procedures...

Reply:
Actually I guess maybe we could make this work with $current_release... Hmm...